spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'i18n'
+
+# Helpers for features
+module FeatureHelpers
+  def t(key)
+    I18n.t(key)
+  end
+
+  def sign_in(user)
+    visit user.instance_of?(User) ? new_user_session_path : new_staff_session_path
```

Comment:
```suggestion
    raise NotImplementedError
```

I suppose you probably grabbed this code from some of your previous projects. That's fine as soon as you remove some project-related code. For now, we don't have neither `new_staff_session_path` nor `new_user_session_path`. So I'd leave `NotImplementedError` here for easier understanding that we need to revisit the place and create implementation if we really want to use the method.